src/gwc/src/main/java/org/geoserver/gwc/config/GeoserverXMLResourceProvider.java

```diff
@@ -23,8 +26,12 @@
 
 public class GeoserverXMLResourceProvider implements ConfigurationResourceProvider {
 
-    private static Log log = LogFactory.getLog(org.geowebcache.config.XMLFileResourceProvider.class);
-    
+    private static Log LOGGER = LogFactory.getLog(org.geowebcache.config.XMLFileResourceProvider.class);
```

Comment:
I know that this was already like this already so it's not your fault, but this looks like a good opportunity for confusing logs (Error occurs in `GeoserverXMLResourceProvider` but is logged as occurring in `XMLFileResourceProvider`)

Not a required change, just a comment.